### PR TITLE
Add mxnet operator (https://github.com/kubeflow/community/issues/136)

### PR DESCRIPTION
**Pull request by suleisl2000**
Add MXNet operator into website repo for doc

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/172)

### CODE REVIEW

**Comment by suleisl2000** [comment]
@googlebot I signed it!

File: content/docs/guides/components/mxnet.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+# MXNet Training
```
Comment (abhi-g) [inline content/docs/guides/components/mxnet.md]: @suleisl2000 You need a header in the markup to ensure hugo can generate corresponding title entries and add it to the navigation etc. Please see the header at some other component guides in this directory for example.

Comment (suleisl2000) [reply content/docs/guides/components/mxnet.md]: @abhi-g Comment is addressed. Please help to check it again.

**Pull request merged**